docs/app/pages/components/sections/pdf-export/pdf-export/snippets/app.ts

```diff
@@ -0,0 +1,251 @@
+import { Component, ViewChild } from '@angular/core';
+import { BaseChartDirective } from 'ng2-charts';
+import { Chart } from 'chart.js';
+import { ColorService, PdfExportService, DashboardOptions } from 'ux-aspects';
+import 'chance';
+
+@Component({
+    selector: 'app',
+    templateUrl: './src/app.component.html',
+    styleUrls: ['./src/app.component.css']
+})
+export class AppComponent {
+
+    @ViewChild(BaseChartDirective) baseChart: BaseChartDirective;
+
+    // configure the directive data
+    barChartData: Chart.ChartData = [{
+        data: [34, 25, 19, 34, 32, 44, 50, 67],
+        borderWidth: 1
+    }];
+
+    barChartLabels: string[] = ['.doc', '.ppt', '.pdf', '.xls', '.html', '.txt', '.csv', '.mht'];
+    barChartOptions: Chart.ChartOptions;
+    barChartLegend: boolean = false;
+    barChartColors: any;
+
+    dataTable = [{
+        document: '.doc',
+        value: 34,
+    }, {
+        document: '.ppt',
+        value: 25,
+    }, {
+        document: '.pdf',
+        value: 19,
+    }, {
+        document: '.xls',
+        value: 34,
+    }, {
+        document: '.html',
+        value: 32,
+    }, {
+        document: '.txt',
+        value: 44,
+    }, {
+        document: '.csv',
+        value: 50,
+    }, {
+        document: '.mht',
+        value: 67,
+    }];
+
+    privacy = [{
+        type: 'Employee data',
+        value: 139
+    }, {
+        type: 'Health data',
+        value: 56
+    }, {
+        type: 'Financial',
+        value: 34
+    }, {
+        type: 'Personal ID',
+        value: 13
+    }, {
+        type: 'Other',
+        value: 2
+    }];
+
+    // configure the directive data
+    donutChart1Data: Chart.ChartData = [{
+        data: [287, 23],
+        borderWidth: 0
+    }];
+
+    donutChart2Data: Chart.ChartData = [{
+        data: [151, 159],
+        borderWidth: 0
+    }];
+
+    donutChartOptions: Chart.ChartOptions;
+    donutChart1Colors: any;
+    donutChart2Colors: any;  
+
+    custodians: string[] = [];
+
+    options: DashboardOptions = {
+        columns: 3,
+        padding: 10,
+        rowHeight: 220,
+        emptyRow: false,
+        minWidth: 187
+    };
+
+    constructor(public colorService: ColorService, private _pdfExportService: PdfExportService) {
```

Comment:
colorService is only used within the constructor so it doesn't need to be added as an instance variable, (make same changes in the non-snippet version of this class too)